Dockerfile

```diff
@@ -5,6 +5,12 @@ RUN apt-get install --no-install-recommends -yq software-properties-common
 RUN apt-get update
 RUN apt-get install --no-install-recommends -y build-essential zlib1g-dev libncurses5-dev libgdbm-dev libnss3-dev libssl-dev libreadline-dev libffi-dev wget curl sudo git
 
+# NOTICE: we need to install SSL 1.1 manually here in order to make BLS command line tools working
+RUN echo "deb http://security.ubuntu.com/ubuntu focal-security main" | sudo tee /etc/apt/sources.list.d/focal-security.list
```

Comment:
From the skale node perspective it's better to use single RUN directive to download all the required dependencies.
Besides there is no point to run apt-get update. 
It also may be a good idea to derive from official node package.

Reply:
I was unable to find good official repo URL do download pre-build libs.